src/lamplib/src/genny/curator.py

```diff
@@ -244,7 +266,7 @@ class CuratorDownloader(Downloader):
     # These build IDs are from the Curator Evergreen task.
     # https://evergreen.mongodb.com/waterfall/curator
 
-    CURATOR_VERSION = "a1293258b30e04ea503e6b91b6b1628789e90093"
+    CURATOR_VERSION = "fa0dfec710dfbe2fb47c15d5800e978717f23614"
```

Comment:
Is this an incidental version bump or required for this?

Reply:
This is needed - I added the Curator command that Genny will be using in this version.